docs/storefront/headless/routes.mdx

```diff
@@ -0,0 +1,204 @@
+# Routes with the GraphQL Storefront API
+
+Routes define specific URL paths for fetching or updating data. The BigCommerce GraphQL Storefront API allows you to retrieve routes for each storefront channel, making it easier to build headless storefront applications using BigCommerce's built-in content. 
+
+When querying a `route`, the response may include the following node types:
+
+* Product
+* Category
+* Brand
+* NormalPage
+* ContactPage
+* RawHtmlPage
+* Blog
+* BlogPost
+
+## Enhanced redirect support
+
+The new `LookUpUrl` capability now supports redirects, helping you determine the appropriate page to display. If a redirect exists for a given path, the `Redirect` node within the `route` query returns the redirect details. Otherwise, it returns `null`.
+
+## Redirect enum values
+
+The `redirectBehavior` argument supports two values:
+
+* **IGNORE**(default): 
+
+   * The Storefront GraphQL API disregards redirects and prioritizes custom URLs. 
+   * If both a redirect and an entity URL exist for the same path, both the `redirect` and `node` nodes return non-null values. 
+   * This behavior ensures backward-compatiblity.
+
+* **FOLLOW**: 
+
+  * For dynamic or association redirects, the `node` node returns the target entity (e.g., category, product, blog, etc.). 
+  * For static/manual redirects, the `node` node returns `null` (since there’s no associated entity), while the `redirect` node provides the redirect details.
+
+## Example queries
+
+### Query: Fetching route details
+
+This example retrieves details about an object (e.g., Product, Brand, or Category) based on a given URL path.
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+    ```graphql filename="Query route details" showLineNumbers copy
+
+     query LookUpUrl(
+       $urlPath: String!
+       # Use GraphQL Query Variables to provide a path
+     ) {
+       site {
+         route(path: $urlPath) {
+           node {
+             __typename
+             id
+             # A different response is returned based on which type of object was matched
+             ... on Category {
+               name
+               description
+             }
+             ... on Brand {
+               name
+               defaultImage {
+                 url(width: 200)
+               }
+             }
+             ... on Product {
+               name
+               description
+               images {
+                 edges {
+                   node {
+                     url(width: 500, height: 500)
+                   }
+                 }
+               }
+             }
+             ... on Redirect {
+               # all redirect attributes
+               toUrl
+             }
+           }
+         }
+       }
+     }
+```
+```json filename="GraphQL variables" showLineNumbers copy 
+    {
+      "urlPath": "/blog/"
```

Comment:
Rather than provide an input URL of `/blog/` I think this example should have an input URL that references one of the entity types that we are defining in the query. Either a product, category, or brand. I might be confusing as to why the query did not define "Blog" as an response type yet that is what was returned.

I do think there is relevance to this type of example but we should clearly describe what is happening. I also don't think it should be the first example. The first example should be successful use of finding a desired entity at a URL.